src/main/java/com/exactpro/th2/act/ActHandler.java

```diff
@@ -441,6 +484,27 @@ private EventID createAndStoreParentEvent(PlaceMessageRequestOrBuilder request,
         }
     }
 
+    private EventID createAndStoreParentEvent(MultiSendRequest request, String actName, Status status) throws IOException {
+        long startTime = System.currentTimeMillis();
+
+        Event event = start()
+                .name(actName + ' ' + request.getMessages(0).getMetadata().getId().getConnectionId().getSessionAlias())
+                .description(request.getDescription())
+                .type(actName)
+                .status(status)
+                .endTimestamp(); // FIXME set properly as is in the last child
```

Comment:
You won't be able to do so with distributed events. I think we can remove that FIXME from here